.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -53,6 +54,49 @@ Bugs & improvements:
 
 - [ ] (1) ...
 
+_____
+
+## Patch Release
+
+### TASKS:
+
+ - [ ] [GIT] Create branch `release/M.m.p` in owncloud/android from `latest`
+ - [ ] [DEV] Update version number and name in build.gradle in owncloudApp module
+ - [ ] [DEV] Update [SBOM](https://cloud.owncloud.com/f/6072870)
+ - [ ] [DIS] Create a folder for the new version like `M.m.p_YYYY-MM-DD` inside the `changelog` folder with the proper fixes
+ - [ ] [DIS] Move the `unreleased` folder from `master` to `changelog` folder, to avoid Calens' conflicts problems
```

Comment:
```suggestion
 - [ ] [DIS] Copy the `unreleased` folder in `master` branch to `changelog` folder in this branch, to avoid Calens' conflicts problems
```
Maybe this tiny change improves understandability

Reply:
👍